### PR TITLE
Home Work 2 done

### PR DESCRIPTION
**Pull request by Ivan-Artemenko**
Home Work 2

### CODE REVIEW

**Comment by markoffden** [comment]
please revise your approach to both tasks
try to solve task using tools JavaScripts provides for such purposes, like for loop etc.

**Comment by Ivan-Artemenko** [comment]
Improved homework_2. I hope you enjoy it.

File: homeworks/Ivan.Artemenko_IvanArtemenko/homework_2/homework_2_1.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  	    let a = '**';	
+        let b = ' ';
+        let j = ((number_1+1)-i);
+        a = a.repeat(i);
+        b = b.repeat(j);
+        console.log(b+a);
+    }    
+}else console.log('You must enter a number from 3 to 15')
+ rl.close();
+});
```
Comment (markoffden) [inline homeworks/Ivan.Artemenko_IvanArtemenko/homework_2/homework_2_1.js]: looks much better :)

File: homeworks/Ivan.Artemenko_IvanArtemenko/homework_2/homework_2_2.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+rl.question('Enter the first number from 500 to 1000 ', (number_1) => {
+	rl.question('Enter the second number from 500 to 1000 ', (number_2) => {
```
Comment (markoffden) [inline homeworks/Ivan.Artemenko_IvanArtemenko/homework_2/homework_2_2.js]: if you ask user input, make sure to add some validation of what user enters in the prompt

File: homeworks/Ivan.Artemenko_IvanArtemenko/homework_2/homework_2_2.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+		let a = number_1-0;
+		let b = number_2-1;
```
Comment (markoffden) [inline homeworks/Ivan.Artemenko_IvanArtemenko/homework_2/homework_2_2.js]: use camelCase pattern, when naming JS variables

**Comment by markoffden** [comment]
also you have some linter issues, pls do the following:

1. pull lates updates from master repo
2. run npm i
3. run npm test
4. fix issues
5. commit changes

**Pull request merged**